### ISSUE TITLE
BUG: NeptuneCallback fails for losses other than BaseLoss

### ISSUE THREAD

**Issue by slawekslex** [neptune-ai/neptune-fastai#22]
### Describe the bug
The line here: https://github.com/neptune-ai/neptune-fastai/blob/b8e0f3c308170861806c7e23423d76685f71f22e/neptune_fastai/impl/__init__.py#L214
assumes that `Learner.loss_func` is an object with a `func` attribute. However `loss_func` can be any callable, like `torch.nn.MSELoss` or just a plain python function.

This causes is to crash with `AttributeError`
### Reproduction
Create a `Learner` by passing `loss_func=nn.MSELoss()` or any other custom function. Together with `NeptuneCallback`
### Expected behavior
Model trains without crashing

**Comment by Blaizzy** [comment]
Hi @slawekslex,

Prince Canuma here, from Neptune.ai,

Thank you for submitting this feedback, I have managed to replicate it with your description and submit a couple of possible fixes to the engineering team, I will let you know once it is fixed.

For now, if this is blocking you. You can edit your local **./neptune_fastai/impl/__init__.py** file line 214 to:

```
@property
def _optimizer_criterion(self) -> str:
    return repr(self.loss_func)
```

Let me know this helps you!

**Comment by slawekslex** [comment]
Thanks for looking into it @Blaizzy. I was able to work around it by setting a dummy attribute on my loss function.
The problem was more in figuring out what's wrong and the general experience of trying to use a library and needing to deal with error after error.

**Comment by slawekslex** [comment]
I'm going to open issues here if I run into anything else. Thanks again for the responsiveness.

**Comment by Blaizzy** [comment]
Most welcome! 👍 

Have great weekend 💯
